simulator/modules/decode/bypass/data_bypass.h

```diff
@@ -23,10 +23,10 @@ class DataBypass
 
     public:
         explicit DataBypass( uint64 long_alu_latency)
-            : last_execution_stage_value( static_cast<uint8>(long_alu_latency - 1))
+            : last_execution_stage_value( narrow_cast<uint8>(long_alu_latency - 1))
```

Comment:
Looks like a bug, long_alu_latency should have Latency type and not uint8. I'll fix that later.